src/Features/Core/Portable/UseAutoProperty/AbstractUseAutoPropertyCodeFixProvider.cs

```diff
@@ -83,14 +86,33 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
 
             var solution = context.Document.Project.Solution;
             var fieldLocations = await Renamer.GetRenameLocationsAsync(
-                solution, SymbolAndProjectId.Create(fieldSymbol, fieldDocument.Project.Id), 
+                solution, SymbolAndProjectId.Create(fieldSymbol, fieldDocument.Project.Id),
                 solution.Options, cancellationToken).ConfigureAwait(false);
 
             // First, create the updated property we want to replace the old property with
             var isWrittenToOutsideOfConstructor = IsWrittenToOutsideOfConstructorOrProperty(fieldSymbol, fieldLocations, property, cancellationToken);
-            var updatedProperty = await UpdatePropertyAsync(propertyDocument, compilation, fieldSymbol, propertySymbol, property,
+            var updatedProperty = await UpdatePropertyAsync(
+                propertyDocument, compilation, fieldSymbol, propertySymbol, property,
                 isWrittenToOutsideOfConstructor, cancellationToken).ConfigureAwait(false);
 
+            // Ensure the new and old property share the same trailing trivia.
+            updatedProperty = updatedProperty.WithTrailingTrivia(property.GetTrailingTrivia());
+
+            // However, if we have a situation where the property is the second member in a type,
+            // and it would become the first, then remove any leading blank lines from it so we
+            // don't have random blanks above it that used to space it from the field that was
+            // there.
+            var nodeToRemove = GetNodeToRemove(declarator);
```

Comment:
why is GetNodeToRemove called twice? could you put this logic after the other call to GetNodeToRemove below?

Reply:
Done.